src/css/custom.css

```diff
@@ -337,6 +337,12 @@ main.max-w-screen-lg {
   font-display: swap;
 }
 
+h2,
+h3 {
+  font-weight: 700;
+  letter-spacing: -2px;
```

Comment:
This is too much reduction I think -
Maybe -1 or -.05 or something?

<img width="476" alt="Screen Shot 2022-07-17 at 9 35 23 AM" src="https://user-images.githubusercontent.com/34380806/179400965-83505dd0-8e79-4786-aecf-6a78956b6fb6.png">